docs/roadmap/roadmap.rst

```diff
@@ -0,0 +1,82 @@
+:title: Roadmap
+:description: The Deis project roadmap
+
+.. _roadmap:
+
+Deis Roadmap
+============
+The Deis Roadmap is a community document created as part of our open :ref:`planning`.
```

Comment:
Replace "our" with "the" and "we deem" with "that are deemed" perhaps? I think we should try not to use first-person pronouns when talking about the Deis project ideally. I think creates semantic distance between the maintainers and the reader.